etc/env.json

```diff
@@ -16,5 +16,8 @@
   "speechNoRespTimeout": 20000,
   "skills": {
     "alarmUri": "cas-alarm-pro.rokid.com"
+  },
+  "trace": {
+    "uploadUrl": "das-tc-service-pro.rokid.com"
```

Comment:
Use `uploadUri` to keep consistent with the `alarmUri`.

Reply:
The URI belongs to the parent class, and the URL belongs to a subclass of the URI.
The difference is that the URI represents the path to the requesting server and defines such a resource. The URL also indicates how to access the resource (http://).